sdl-core/src/main/scala/io/smartdatalake/app/GlobalConfig.scala

```diff
@@ -58,6 +58,13 @@ import org.apache.spark.util.PrivateAccessor
  * @param synchronousStreamingTriggerIntervalSec Trigger interval for synchronous actions in streaming mode in seconds (default = 60 seconds)
  *                       The synchronous actions of the DAG will be executed with this interval if possile.
  *                       Note that for asynchronous actions there are separate settings, e.g. SparkStreamingMode.triggerInterval.
+ * @param allowAsRecursiveInput allow exception for specific DataObjects to the validation rules for Action.recursiveInputIds.
+ *                              The validation rules are
```

Comment:
```suggestion
 *                              The DataObjects in the allowAsRecursiveInput List must fullfil the following validation rules:
```

Reply:
I think that with the modifications above and below it is clearer as is.